integration/monai/examples/spleen_ct_segmentation_local/README.md

```diff
@@ -139,36 +150,36 @@ For details about resource management and consumption, please refer to the [docu
 > To speed up your experimentation, you can reduce the `num_rounds` value in `config_fed_server.json`, e.g. to 5 rounds.
 
 ### 5.1 Experiment tracking with MLflow
-Using MONAI's experiment [tracking feature](https://github.com/Project-MONAI/tutorials/tree/main/experiment_management),
-we can use MLflow to track the model performance on the local clients.
+Experiment tracking with MLflow in version 2.4.0 now uses `NVFlareStatsHandler` to stream events
```

Comment:
Should we mention version numbers in our READMEs?

Reply:
I can remove it